sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -669,6 +716,232 @@ export class BlobServiceClient extends StorageClient {
     }
   }
 
+  /**
+   * The Filter Blobs operation enables callers to list blobs across all containers whose tags
+   * match a given search expression. Filter blobs searches across all containers within a
+   * storage account but can be scoped within the expression to a single container.
+   *
+   * @private
+   * @param {string} tagFilterSqlExpression The where parameter enables the caller to query blobs whose tags match a given expression.
+   *                                        The given expression must evaluate to true for a blob to be returned in the results.
+   *                                        The[OData - ABNF] filter syntax rule defines the formal grammar for the value of the where query parameter;
```

Comment:
What about link [https://docs.microsoft.com/en-us/azure/storage/blobs/storage-manage-find-blobs?tabs=azure-portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-manage-find-blobs?tabs=azure-portal)

Reply:
That's great! Updated.